proto/hypertrace/agent/config/v1/config.proto

```diff
@@ -63,6 +63,9 @@ message Reporting {
 
     // reporter type for metrics.
     MetricReporterType metric_reporter_type = 8;
+
+    // When `true`, modifies grpc resolver to use dns instead of passthrough and configure round robin client side loadbalancing
```

Comment:
We said it's fine to have this on by default, right? i.e no need for this config. Unless you want a way to turn it off.

Reply:
Yeah, I still think the go-agent change is safe. my concern is largely with unknown/unexpected deployment configurations & not having a nice way to roll things back if anything unexpected does happen.